Changelog.rst

```diff
@@ -85,15 +283,49 @@ A simple find and replace of ``concat_fragment`` and ``concat_build`` in legacy
 code with ``simpcat_fragment`` and ``simpcat_build`` should suffice. Be sure to
 check for ``Concat_fragment`` and ``Concat_build`` resource dependencies!
 
-pupmod-simp-pupmod
-""""""""""""""""""
+pupmod-simp-simplib
+"""""""""""""""""""
 
-This module has code that only works with Puppet 4
+Removed all manifests and Puppet code from this module. It now only contains
+functions and custom type aliases.
 
-pupmod-simp-foreman
-"""""""""""""""""""
+List of modules that were created or forked after removing content from simplib:
+
+* pupmod-simp-at
+* pupmod-simp-chkrootkit
+* pupmod-simp-useradd
+* pupmod-simp-swap
+* pupmod-simp-cron
+* pupmod-simp-resolv
+* pupmod-simp-issue
+* pupmod-simp-fips
+* puppetlabs-motd
+* trlinkin-nsswitch
+* camptocamp-kmod
+* puppetlabs-motd
+* saz-timezone
+
+The rest of the content was added to our profile module, simp-simp.
+
+pupmod-simp-snmpd
+"""""""""""""""""
+
+The ``snmpd`` module has been removed until updates can be made available.
+
+pupmod-simp-sudo
+""""""""""""""""
+
+Added method to create user_specification resources through hiera.
+
+pupmod-simp-xwindows
+""""""""""""""""""""
+
+Module has been rewritten and renamed to pupmod-simp-gdm.
+
+ELG Stack
+"""""""""
 
-The Foreman module has been removed until it works consistently with Puppet 4
+pupmod-simp-logstash now uses TLS.
 
 Puppet AIO
```

Comment:
This is also referenced at the top, and should probably be moved up there for consistency's sake